apidocs/cloud-api-source/source/index.md

```diff
@@ -97,6 +97,12 @@ Examples:
 - The user wants to operate on the node cluster list endpoint in an organization called `exampleorg` (which they have permission to see):
     - https://cloud.docker.com/api/infra/v1/exampleorg/nodecluster/
 
+### Namespaced endpoints in the docker-cloud CLI
+
+If you are using namespaces with the `docker-cloud` CLI, set them by changing
+the value of the `DOCKERCLOUD_NAMESPACE` environment variable. You can either
+set this globally, or specify it before each CLI command.
+
```

Comment:
I would reference the github README the same way https://github.com/docker/docker.github.io/pull/876/files#diff-7680652211f2a1fc8963e632c69a147bR102